### DISCUSSION TITLE
How to exit Yazi to the current path?

### DISCUSSION THREAD

**Discussion by luiz00martins** [sxyazi/yazi#418] [1. Q&A] [answered]
Note that I'm not talking about opening a shell inside yazi, but actually exiting yazi, and then going to the last yazi path.

It seems like the links provided by [this comment](https://github.com/sxyazi/yazi/issues/177#issuecomment-1724185357) and [this answer](https://github.com/sxyazi/yazi/discussions/242#discussioncomment-7174020) don't work anymore.

**Comment by sxyazi** [comment, marked as answer]
Are you looking for https://yazi-rs.github.io/docs/quick-start#changing-working-directory-when-exiting-yazi?

**Reply by luiz00martins** [reply]
Yep, that's it! Should probably have one for windows there too. This works for me:
```pwsh
function ya {
    $tmp = [System.IO.Path]::GetTempFileName()
    yazi --cwd-file="$tmp"
    $cwd = Get-Content -Path $tmp
    if (-not [String]::IsNullOrEmpty($cwd) -and $cwd -ne $PWD.Path) {
        Set-Location -Path $cwd
    }
    Remove-Item -Path $tmp
}

```

**Reply by sxyazi** [reply]
Glad to hear that.

Is this PowerShell? Would you like to create a PR to add it to the documentation? (Click "Edit this page" at the bottom of that doc.)

**Reply by luiz00martins** [reply]
Done :)

https://github.com/yazi-rs/yazi-rs.github.io/pull/20

**Reply by sxyazi** [reply]
Thank you!
